### PR TITLE
Milestone 4: Contact-form

### PR DESCRIPTION
**Pull request by Xghayor**
In this milestone i completed:

1.Added a contact form
2.Added a footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -232,6 +232,42 @@ <h3> <img src="./img/Rectangle 57.png" alt="points" width="10%"> skills</h3>
...
+    </div>
+    <div class="form-data">
+      <form action="https://formspree.io/f/xlekyljn" method="POST">
+          <input type="text" class="first-input" placeholder="First Name" maxlength="30" required>
+          <input type="text"  class="second-input"  placeholder="Last Name" maxlength="30" required> <br>
+          <input  type="email"  class="third-input"  id="input4"  placeholder="Email Address"  required> <br>
+          <textarea type="text" class="fourth-input"  maxlength="500" name="message"  placeholder="Write me something..." required ></textarea> <br>
+        <button type="submit" class="form-btn">Get in touch</button>
+      </form>
+    </div>
```
Comment (Tufahel) [inline index.html]: - ***Your Design***
image

- [ ] Here I can see you have used two input fields for `name`, as `first name` and `last name`. But I can see in the given template that only the `Full Name` input field is used. Kindly fix this issue.

- ***Given Design***
image

File: index.html [hunk 2]
```diff
@@ -232,6 +232,42 @@ <h3> <img src="./img/Rectangle 57.png" alt="points" width="10%"> skills</h3>
...
+<footer>
+  <div>
+    <ul class="foter-list">
+      <li><a href="#"><img src="./img/footer/1.png" alt="angel"></a></li>
+      <li><a href="#"><img src="./img/footer/2.png" alt="github"></a></li>
+      <li><a href="#"><img src="./img/footer/3.png" alt="medium"></a></li>
+      <li><a href="#"><img src="./img/footer/4.png" alt="twitter"></a></li>
+      <li><a href="#"><img src="./img/footer/5.png" alt="vector"></a></li>
+    </ul>
+  </div>
```
Comment (Tufahel) [inline index.html]: - ***Given Design***
image

- For icons, kindly consider the things below to match the given design:
- [ ] Give more margin between the icons.
- [ ] Give more padding/margin between `hr` tag and `footer`
- [ ] Maintain the serial of icons as per the given design.

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Xghayor, @ausiemass  :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Descriptive pull request.
✔️ Follow the correct GitHub flow.
✔️ Commit messages are meaningful.
✔️ Form submission is working fine.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 3]
```diff
@@ -433,6 +453,118 @@ hr {
...
+}
+
+.foter-list {
+  display: flex;
+  gap: 20px;
+  list-style: none;
+  padding: 15px;
+  margin: 5px;
+  color: #505f79;
+}
```
Comment (franclobo) [inline styles.css]: As required by the previous CR: Maintain the serial of icons as per the given design. I suggest adding `flex-flow: row-reverse;` to give the correct order to the social icons to be similar to the Figma design.

| Figma template | Your design | 
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Xghayor, 

# ♻️ Required Changes ♻️

istockphoto-1079018918-612x612

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Correct contact form ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @Xghayor, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights 🎯
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**